packages/editor-ui/src/components/mixins/nodeHelpers.ts

```diff
@@ -44,8 +43,8 @@ export const nodeHelpers = mixins(
 			...mapGetters('credentials', [ 'getCredentialTypeByName', 'getCredentialsByType' ]),
 		},
 		methods: {
-			isHttpRequestNodeV2 (node: INodeUi): boolean {
-				return node.type === HTTP_REQUEST_NODE_TYPE && node.typeVersion === 2;
+			hasProxyAuth (node: INodeUi): boolean {
+				return ['nodeCredentialType', 'genericAuthType'].some(t => Object.keys(node.parameters).includes(t));
```

Comment:
just wondering, why did we circle back to this?

Reply:
Reverted after refactoring generic auth as `credentialsSelect`